cbits/decaf/p448/f_field.h

```diff
@@ -27,7 +27,7 @@
 #define SER_BYTES 56
 typedef struct cryptonite_gf_448_s {
     word_t limb[NLIMBS];
-} __attribute__((aligned(16))) cryptonite_gf_448_s, cryptonite_gf_448_t[1];
+} VECTOR_ALIGNED cryptonite_gf_448_s, cryptonite_gf_448_t[1];
```

Comment:
Observed a general protection fault at `vmovdqa (%r15),%ymm2` when `r15` held the value `0x7fffffff53d0`. 16 byte alignment is not sufficient for AVX2 (needs to be 32).